implementation/elasticsearch/src/main/java/net/opentsdb/meta/impl/es/ESMetaResponse.java

```diff
@@ -737,6 +738,10 @@ NamespacedAggregatedDocumentResult parseTimeseries(
         result.addTimeSeries(id, meta_query, metric, false);
       }
     }
+    
+    if (isMultiGet && result.timeSeries().isEmpty()) {
```

Comment:
Can the result be null at this point ?

Reply:
Nope, safe here as there's a null check in the getter.